src/app/navigation/utils.ts

```diff
@@ -1,5 +1,6 @@
 import { flatten, unflatten } from 'flat';
 import { Action, Location } from 'history';
+import { isNull, omitBy } from 'lodash';
```

Comment:
also i prefer `lodash/fp`, which auto-curries the functions and allows you to write things like
```
import isNull from 'lodash/fp/isNull';
import omitBy from 'lodash/fp/omitBy';

const omitByNull = omitBy(isNull);

const filteredSearch = omitByNull(search);
```

not that it really matters a lot in this case but consistency is nice